rocAL/rocAL/include/augmentations/geometry_augmentations/node_random_crop.h

```diff
@@ -25,12 +25,11 @@ THE SOFTWARE.
 #include "parameter_factory.h"
 #include "parameter_crop_factory.h"
 
-class RandomCropNode : public Node
-{
+class RandomCropNode : public Node {
 public:
     RandomCropNode(const std::vector<Tensor *> &inputs, const std::vector<Tensor *> &outputs);
     RandomCropNode() = delete;
-    void init(float area, float aspect_ratio, float x_drift, float y_drift);
+    ~RandomCropNode();
```

Comment:
we don't need destructor here unless you want to do something special here. Otherwise the base class destructor will be called

Reply:
The destructors are added in all the crop related nodes.
The crop coordinates needs to be passed as vx_tensor hence we need to create a vx_tensor in the node and release it.
Hence the destructor is introduced in all crop related nodes to release the vx_tensor for the crop

Reply:
Added destructor only in CropNode will be used by other derived crop related augmentations like RandomCropNode, CropMirrorNormalizeNode etc